ui/stories/index.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import Hello from '../src/component/hello';
+import BarComponent from '../src/component/bar';
+
+const source = [
+  ['product', '2015', '2016', '2017'],
+  ['Matcha Latte', 43.3, 85.8, 93.7],
+  ['Milk Tea', 83.1, 73.4, 55.1],
+  ['Cheese Cocoa', 86.4, 65.2, 82.5],
+  ['Walnut Brownie', 72.4, 53.9, 39.1],
+];
 
 storiesOf('Hello', module)
   .add('hello', () => (
     <Hello />
   ));
 
+storiesOf('Bar Chart', module)
+  .add('Simple bar', () => (
+    <BarComponent source={source} />
```

Comment:
My 2 cent: Since echart accepts `option` object which not only contains data source but also tooltip, legend, title. etc.
It would be more straight forward to pass `option` as a prop of BarComponent

Reply:
This is the first step of chart component, accepting source and render chart. For the other options, we can add them when we have specified&clear requirements. 

As currently we only have source, just name it ```source```.